### PR TITLE
add charts and testimonial

### PR DESCRIPTION
**Pull request by boyd-nguyen**
Review link: http://127.0.0.1:5000/case-studies/critical-race-theory/

### CODE REVIEW

File: Australian Digital Observatory/content/case-studies/critical-race-theory/crt_1.png
Comment (Quiet27) [inline Australian Digital Observatory/content/case-studies/critical-race-theory/crt_1.png:1]: Is this image meant to appear on the CRT case study page?

**Review by Quiet27 — APPROVED** [review]
Thanks @boyd-nguyen! Another great case study. One thing I noticed was that one of the charts wasn't appearing on the CRT case study page. I wasn't sure if this was intentional, so I've approved the PR anyway 😸

File: Australian Digital Observatory/content/case-studies/critical-race-theory/crt_1.png
Comment (boyd-nguyen) [reply Australian Digital Observatory/content/case-studies/critical-race-theory/crt_1.png:1]: I added both images that Naomi sent through for our records. But the styling between them is different so I only displayed one. Thanks for checking and approving.

Comment (Quiet27) [reply Australian Digital Observatory/content/case-studies/critical-race-theory/crt_1.png:1]: Awesome, thanks!

**Pull request merged**